packages/oui-navbar/src/brand/navbar-brand.component.js

```diff
@@ -22,6 +25,16 @@ export default {
                     .removeAttr("aria-label")
             );
         }
+
+        onNavigateLink () {
+            if (angular.isFunction(this.onClick)) {
+                this.$q.when(this.onClick()).finally(() => {
```

Comment:
I think this part should be in the Manager side. It seems a bit complex for just adding a click function on a link :)

Normally, `ng-click="$ctrl.onClick()"` should be enough. We could even pass the event parameter if needed like  `ng-click="$ctrl.onClick($event)"`.